docsearch/sidebars.js

```diff
@@ -7,9 +7,16 @@
 
 module.exports = {
   docs: {
-    'Getting Started': ['what-is-docsearch', 'how', 'apply'],
-    'Adding DocSearch to your UI': ['dropdown', 'styling', 'behavior'],
-    'Requirements, tips, FAQ': ['requirements','tips','faq','integrations'],
-    'Let\'s Get Technical': ['scraper','engine','config','css-helper','index','run-your-own']
-  },
+    "Getting Started": ["what-is-docsearch", "how", "who-can-apply", "apply"],
+    "Adding DocSearch to your UI": ["dropdown", "styling", "behavior"],
+    "Requirements, tips, FAQ": ["requirements", "tips", "faq", "integrations"],
+    "Let's Get Technical": [
```

Comment:
rename to "under the hood" ?

Reply:
Good idea. We used to have this label in the past, I will put it back